File: Qualify the round.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,a,b;
        scanf("%d%d%d",&x,&a,&b);
        if(x<=a+b*2)
        printf("Qualify\n");
        else
        printf("NotQualify\n");
    }
}

File: Qualify the round.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int a=sc.nextInt();
            int b=sc.nextInt();
            if(a+(b*2)>=x)
            System.out.println("Qualify");
            else
            System.out.println("NotQualify");
        }
    }
}